src/test/javascript/spec/component/overview/course-faq/course-faq.component.spec.ts

```
@@ -18,12 +18,13 @@ import { CustomExerciseCategoryBadgeComponent } from 'app/shared/exercise-catego
 import { CourseFaqAccordionComponent } from 'app/overview/course-faq/course-faq-accordion-component';
 import { Faq } from 'app/entities/faq.model';
 import { FaqCategory } from 'app/entities/faq-category.model';
+import { SearchFilterComponent } from 'app/shared/search-filter/search-filter.component';
 
 function createFaq(id: number, category: string, color: string): Faq {
     const faq = new Faq();
     faq.id = id;
-    faq.questionTitle = 'questionTitle';
-    faq.questionAnswer = 'questionAnswer';
+    faq.questionTitle = 'questionTitle ' + id;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)